package.json

```diff
@@ -30,8 +30,8 @@
 		"typescript": "5.5.x"
 	},
 	"scripts": {
-		"format": "prettier --write \"./{examples,lib,test}/**/*.ts\"",
-		"lint": "prettier --write \"./{examples,lib,test}/**/*.ts\" && eslint \"./{examples,lib,test}/**/*.ts\" --fix",
+		"format": "prettier --write \"**/*.ts\"",
+		"lint": "npm run format && eslint --fix",
```

Comment:
The `lint` script is no longer correct: this used to verify the linting and **not** format & fix issues!
The `lint` command is also used in the GitHub action `code-format` action to verify if everything is correct:
- Original command in the JS version: `prettier --check "./{examples,lib,test}/**/*.js" index.js && eslint . --ext .js,.jsx,.ts,.tsx`
- If something was not according to the format definition, `prettier --check` & `eslint` output the issues and exit with an exit code > 0.
- `prettier --write` does **not** return with exit code > 0 if it reformats some files!
- A non-zero exit code fails the GitHub action and informs the developer about format / linting issues

What we can change to make it more clear:
- change `lint` script to behave similar to `format`: only run eslint with the --fix option
- add `code-check` which resembles the original command: runs `prettier --check ...` and `eslint` without the --fix option.
  - use the new `code-check` command in the GitHub action.

Reply:
I added the modifications you mentioned above